_partials/self-hosted/_smtp.mdx

```diff
@@ -0,0 +1,54 @@
+---
+partial_category: self-hosted
+partial_name: smtp
+---
+
+You can configure a Simple Mail Transport Protocol (SMTP) server to send emails on behalf of your {props.version}
+instance. An SMTP server is required to change the email address of the admin user and to send the initial tenant
+invitation email.
```

Comment:
I'm not sure I fully understand this sentence. Why is an SMTP server required to change an admin email address?

Does this need more context or is it obvious to someone configuring this?